pkg/internal/ebpf/tctracer/tctracer.go

```diff
@@ -68,39 +67,10 @@ func (p *Tracer) Load() (*ebpf.CollectionSpec, error) {
 }
 
 func (p *Tracer) SetupTailCalls() {
-	for _, tc := range []struct {
-		index int
-		prog  *ebpf.Program
-	}{
-		{
-			index: 0,
-			prog:  p.bpfObjects.BeylaPacketExtenderWriteMsgTp,
-		},
-	} {
-		err := p.bpfObjects.ExtenderJumpTable.Update(uint32(tc.index), uint32(tc.prog.FD()), ebpf.UpdateAny)
-
-		if err != nil {
-			p.log.Error("error loading info tail call jump table", "error", err)
-		}
-	}
 }
 
 func (p *Tracer) Constants() map[string]any {
-	m := make(map[string]any, 2)
-
-	m["wakeup_data_bytes"] = uint32(p.cfg.EBPF.WakeupLen) * uint32(unsafe.Sizeof(ebpfcommon.HTTPRequestTrace{}))
```

Comment:
I think we don't need this for now in the packet extender, but it would be a nasty bug to track if ever needed to send an event from the tpinjector on the ring buffer. Maybe we should port this or make a common thing all programs call.